crates/driver/src/domain/competition/solution/mod.rs

```diff
@@ -179,7 +187,11 @@ impl Solution {
                 trade.order().side,
                 executed,
                 custom_prices,
-                trade.order().protocol_fees.clone(),
+                if self.fee_handler == FeeHandler::Driver {
+                    trade.order().protocol_fees.clone()
+                } else {
+                    vec![]
+                },
```

Comment:
This seems wrong to me. Why would the fees not count for the scoring if the driver withholds them themselves? This would mean drivers that withhold fees would have a disadvantage in the scoring over solvers that don't (which is not something we want to do).

The score should be the same in both cases.

Reply:
Oh, my bad, I thought we also needed to remove it from the score, fixing it right now!
https://github.com/cowprotocol/services/pull/2569